e2e/rbd.go

```diff
@@ -1188,7 +1210,15 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to create storageclass with error %v", err)
 				}
 
-				validatePVCClone(1, pvcPath, appPath, pvcSmartClonePath, appSmartClonePath, noKMS, isThickPVC, f)
+				validatePVCClone(1,
+					pvcPath,
+					appPath,
+					pvcSmartClonePath,
+					appSmartClonePath,
+					"",
```

Comment:
as a readability improvement, consider defining `noDataPool` as a `""`, similar to `noKMS`

Reply:
Updated :+1: